[hplip] Unable to build

```
/usr/local/bin/ld: skipping incompatible ./prnt/hpcups//libImageProcessor.so when searching for -lImageProcessor
/usr/local/bin/ld: cannot find -lImageProcessor
collect2: error: ld returned 1 exit status
make: *** [Makefile:5285: hpcups] Error 1
make: *** Waiting for unfinished jobs....
mv -f .deps/dat2drv-Dat2drv.Tpo .deps/dat2drv-Dat2drv.Po
mv -f .deps/locatedriver-FindPPD.Tpo .deps/locatedriver-FindPPD.Po
hplip failed to build: `make V=0 -j4` exited with 2
```
This is likely due to a missing library for armv7l.

On my Samsung Chromebook Plus v1 (ARM) I get the following:

```
chronos@localhost /usr/local/lib/crew/packages $ crew install -s hplip
hplip: Print, Scan and Fax Drivers for Linux
https://developers.hp.com/hp-linux-imaging-and-printing/
version 3.18.7
Downloading source...
############################################################################################################################################################### 100.0%############################################################################################################################################################### 100.0%############################################################################################################################################################### 100.0%############################################################################################################################################################### 100.0%############################################################################################################################################################### 100.0%
Archive downloaded
Unpacking archive using 'tar', this may take a while...
gcc (GCC) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking for g++... g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking for style of include used by make... GNU
checking dependency style of g++... gcc3
checking for gcc... gcc
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking dependency style of gcc... gcc3
checking build system type... ./config.guess: line 1438: /tmp/cgBT2trZ/dummy: Permission denied
./config.guess: unable to guess system type

This script, last modified 2011-05-11, has failed to recognize
the operating system you are using. It is advised that you
download the most up to date version of the config scripts from

  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
and
  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

If the version you run (./config.guess) is already up to date, please
send the following data and any information you think might be
pertinent to <config-patches@gnu.org> in order to provide the needed
information to handle your system.

config.guess timestamp = 2011-05-11

uname -m = aarch64
uname -r = 4.4.178-16505-g1c7029432a2d
uname -s = Linux
uname -v = #1 SMP PREEMPT Sat Jul 13 23:34:03 PDT 2019

/usr/bin/uname -p = ARMv8 Processor rev 4 (v8l)
/bin/uname -X     = 

hostinfo               = 
/bin/universe          = 
/usr/bin/arch -k       = 
/bin/arch              = 
/usr/bin/oslevel       = 
/usr/convex/getsysinfo = 

UNAME_MACHINE = aarch64
UNAME_RELEASE = 4.4.178-16505-g1c7029432a2d
UNAME_SYSTEM  = Linux
UNAME_VERSION = #1 SMP PREEMPT Sat Jul 13 23:34:03 PDT 2019
configure: error: cannot guess build type; you must specify one
hplip failed to install: `./configure --disable-network-build --prefix=/usr/local --libdir=/usr/local/lib` exited with 1
```

--- a/packages/hplip.rb
+++ b/packages/hplip.rb
@@ -4,12 +4,19 @@ class Hplip < Package
   description 'Print, Scan and Fax Drivers for Linux'
   homepage 'https://developers.hp.com/hp-linux-imaging-and-printing/'
   version '3.20.2'
-  source_url 'https://sourceforge.net/projects/hplip/files/hplip/3.20.2/hplip-3.20.2.tar.gz'
-  source_sha256 '90c49d74688b4d745a739a6db9bf8dbdfa134c24e921e31909bffe9d84f471c2'
+  case ARCH
+  when 'i686', 'x86_64'
+    source_url 'https://sourceforge.net/projects/hplip/files/hplip/3.20.2/hplip-3.20.2.tar.gz'
+    source_sha256 '90c49d74688b4d745a739a6db9bf8dbdfa134c24e921e31909bffe9d84f471c2'
+  end
 
   binary_url ({
+      i686: 'https://dl.bintray.com/chromebrew/chromebrew/hplip-3.20.2-chromeos-i686.tar.xz',
+    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/hplip-3.20.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+      i686: '3665d159fe1684d280689e09546a9f6cb0ab7be68d4e670f40c4111ae015d8b7',
+    x86_64: '05b80f04ea8ac68ffad990ed86140932f6353d25f7fd116df5190e8484385f09',
   })
 
   depends_on 'cups'

--- a/packages/hunspell_base.rb
+++ b/packages/hunspell_base.rb
@@ -8,8 +8,16 @@ class Hunspell_base < Package
   source_sha256 'bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell_base-1.7.0-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell_base-1.7.0-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell_base-1.7.0-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell_base-1.7.0-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'e3e77aa133961b491fee9c7b139d3ae40f04a407faff775cf76a204a947a7b62',
+     armv7l: 'e3e77aa133961b491fee9c7b139d3ae40f04a407faff775cf76a204a947a7b62',
+       i686: 'd4cdc61aac3ab02d3234146364cb1c747e0c070a219197f6990f1d61877b3b13',
+     x86_64: '9f7a88d1e448094147b004a80b62ec5e9cf25b5e93ccbb363a2dddb3461ed8a2',
   })
 
   depends_on 'readline'

--- a/packages/qrencode.rb
+++ b/packages/qrencode.rb
@@ -8,8 +8,16 @@ class Qrencode < Package
   source_sha256 'dbabe79c07614625d1f74d8c0ae2ee5358c4e27eab8fd8fe31f9365f821a3b1d'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/qrencode-4.0.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/qrencode-4.0.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/qrencode-4.0.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/qrencode-4.0.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '1135a62562f8a523968b5affb3fbd32cf1068701de6024ab1d1b612b39065cd3',
+     armv7l: '1135a62562f8a523968b5affb3fbd32cf1068701de6024ab1d1b612b39065cd3',
+       i686: 'a71bbdf13f9a57930d3d0064becdaa57613d3e16fe446e40698bc377881c9175',
+     x86_64: '20d0f46764c934a1d32f114c0488fb9b79a1761dbb757d51a009a2547b3cd682',
   })
 
 

--- a/packages/testdisk.rb
+++ b/packages/testdisk.rb
@@ -8,8 +8,16 @@ class Testdisk < Package
   source_sha256 'c95dd532dad353713e8ca895a3faac31acef284f9f0fad299f69181fec583313'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/testdisk-7.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/testdisk-7.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/testdisk-7.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/testdisk-7.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'd1d42ff50e0792de2c5ddaec3abcb5de9e7057a3d7f63f883cdf283d336850ce',
+     armv7l: 'd1d42ff50e0792de2c5ddaec3abcb5de9e7057a3d7f63f883cdf283d336850ce',
+       i686: 'a3da5c747e406e52ed72d119ba4c308e6465c85b594cf21be9e9e63151551bc7',
+     x86_64: '8b5a399c8e43c821156eb632805fe94f0d6f81e99aaffbeb2e54f54e3a887c77',
   })
 
   depends_on 'apriconv'

